cmd/dev.go

```diff
@@ -0,0 +1,158 @@
+//go:build !windows
+
+package cmd
+
+import (
+	"fmt"
+	"github.com/pkg/errors"
+	"github.com/skatteetaten/ao/pkg/auroraconfig"
+	"github.com/skatteetaten/ao/pkg/deploymentspec"
+	"github.com/skatteetaten/ao/pkg/prompt"
+	architect "github.com/skatteetaten/architect/v2/pkg/build"
+	architectConfig "github.com/skatteetaten/architect/v2/pkg/config"
+	"github.com/spf13/cobra"
+	"os"
+	"strings"
+)
+
+const OpenshiftRegistryFormat = "default-route-openshift-image-registry.apps.%s.paas.skead.no"
+
+var (
+	devCmd = &cobra.Command{
+		Use:         "dev",
+		Short:       "Perform dev operations",
+		Annotations: map[string]string{"type": "development"},
+	}
+
+	rolloutCmd = &cobra.Command{
+		Use:   "rollout <applicationdeploymentRef> <Leveransepakke file>",
+		Short: "Rollout leveransepakke to openshift for development type",
+		RunE:  Rollout,
+	}
+)
+
+func init() {
+	devCmd.AddCommand(rolloutCmd)
+	RootCmd.AddCommand(devCmd)
+}
+
+func Rollout(cmd *cobra.Command, args []string) error {
+	if len(args) != 2 {
+		return cmd.Usage()
+	}
+
+	leveransePakkeFile := args[1]
+	if err := verifyFileExists(leveransePakkeFile); err != nil {
+		return err
+	}
+
+	fileNames, err := DefaultAPIClient.GetFileNames()
+	if err != nil {
+		return err
+	}
+
+	search := args[0]
+
+	matches, err := getFuzzyMatchedResult(search, fileNames)
+	if err != nil {
+		return err
+	}
+
+	spec, err := getSpec(matches)
+	if err != nil {
+		return err
+	}
+
+	deployType := spec.Get("type")
+	if deployType != "development" {
+		return fmt.Errorf("You need to specify type=development in AuroraConfig. Current type=%s", deployType)
+	}
+
```

Comment:
Ville ventet med prompt om "Do you wish to rollout ...." til man kommer hit.  
Det er blir litt lite tilfredstillende brukerflyt å be om bekreftelse og så feile fordi man sjekker type= development etterpå.

Reply:
Ser prompten nå er plassert i getSpec istedenfor getFuzzyMatchedAdr.  Fortsatt kommer dermed sjekken av om type = "development" etter at spørsmålet er stilt.